### ISSUE TITLE
[Train] Port over `timm` example to Train

### ISSUE THREAD

**Issue by amogkam** [ray-project/ray#21020]
In the effort to have more realistic examples for Ray Train, we should port over the `timm` example (like what we had for Ray SGD v1 https://github.com/ray-project/ray/tree/master/python/ray/util/sgd/torch/examples/image_models) and also show how to do more advanced training logic such as exponential moving average or sync batchnorm.

Also showing a performance comparison between Datasets+Train vs. `timm` would be cool.

**Comment by richardliaw** [comment]
This might be a great thing to cover as part of our imagery story in AIR...

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
